modules/formulize/include/functions.php

```diff
@@ -4873,6 +4873,11 @@ function buildConditionsFilterSQL($conditions, $targetFormId, $curlyBracketEntry
                 } else {
                     $conditionsfilter .= " AND ";
                 }
+
```

Comment:
This block and the next one... where is $newFilterOps coming from? I think this code is nonfunctional, or meant to do something different from what the current code makes possible at this point.

Reply:
Does the _buildConditionsFilterSQL return different values in the UCOSP branch perhaps?

Reply:
Yup, there are some quite significant changes for _buildConditionsFilterSQL between the current one and UCOSP branch. We can talk about what to do here?

Reply:
Yup, there are some quite significant changes for _buildConditionsFilterSQL between the current one and UCOSP branch. We can talk about what to do here?

Reply:
pulled out the newFiltersOps for now